<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreFilm</class>
 <widget class="QDialog" name="FenetreFilm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Vue</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="w_vue_oui">
        <property name="text">
         <string>Oui</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="w_vue_non">
        <property name="text">
         <string>Non</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Support</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QCheckBox" name="w_dvd">
        <property name="text">
         <string>DVD</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="w_br">
        <property name="text">
         <string>Blue-ray</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="w_graver">
        <property name="text">
         <string>Graver</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="w_divx">
        <property name="text">
         <string>DivX</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="w_vhs">
        <property name="text">
         <string>VHS</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="l_acteur">
       <property name="text">
        <string>Acteurs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="w_acteur">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="w_nom">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="l_type">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="w_type">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="l_sortie">
         <property name="text">
          <string>Sortie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="w_sortie">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="l_duree">
         <property name="text">
          <string>Durée</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="w_duree">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="l_real">
         <property name="text">
          <string>Réalisateur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="w_real">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="l_genre">
         <property name="text">
          <string>Genre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="w_genre">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="l_scenar">
         <property name="text">
          <string>Scenariste</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="w_scenar">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>w_vue_oui</tabstop>
  <tabstop>w_vue_non</tabstop>
  <tabstop>w_dvd</tabstop>
  <tabstop>w_graver</tabstop>
  <tabstop>w_divx</tabstop>
  <tabstop>w_br</tabstop>
  <tabstop>w_vhs</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>w_type</tabstop>
  <tabstop>w_sortie</tabstop>
  <tabstop>w_genre</tabstop>
  <tabstop>w_duree</tabstop>
  <tabstop>w_scenar</tabstop>
  <tabstop>w_real</tabstop>
  <tabstop>w_acteur</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FenetreFilm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FenetreFilm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
